.github/workflows/companion.yml

```diff
@@ -46,4 +46,23 @@ jobs:
         run: corepack yarn workspaces focus @uppy/companion
       - name: Run tests
         run: corepack yarn run test:companion
+      - name: Run type checks in focused workspace
+        run: corepack yarn run build:companion
 
+  docker:
+    name: Build docker
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_BUILDKIT: 0
+      COMPOSE_DOCKER_CLI_BUILD: 0
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v3
+      - uses: docker/setup-qemu-action@v3
+      - uses: docker/setup-buildx-action@v3
+      - name: Build docker image
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          platforms: linux/amd64,linux/arm64
+          file: Dockerfile
```

Comment:
I think we should revert this. Not worth it to run this on every PR change
```suggestion
```

Reply:
I thought we aligned on only uploading to docker on release?

Reply:
This is not upload, only build docker

Reply:
Alright let's remove it then indeed